.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  check:
+    name: Execute test, check style, lint...
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.10.6
+          cache: pip
+      - name: Install tox
+        run: |
+          python -m pip install -U pip
+          pip install tox
+      - name: Test application with tox
+        run: |
+          tox
```

Comment:
Isn't possible to make `tox` return a JUnit-compatible artifact, which can then be exported from GitHub using Artifacts?